### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by StasOkhrym**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,49 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str,):
```
Comment (mltdll) [inline app/main.py:3]: You have a trailing comma here, remove it.

**Review by mltdll — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (StasOkhrym) [reply app/main.py:3]: Yes, sir!

**Review by mltdll — APPROVED** [review]

**Review by mltdll — APPROVED** [review]
Wow, my approve was so fast it turned out to be pre-emptive. I'll try not to check first from now on.

**Review by Nattalli — APPROVED** [review]
Good job!

**Pull request closed without merging**